### ISSUE TITLE
Remove fiber.create(), fiber.resume(), fiber.detach(). Make status() a method

### ISSUE THREAD

**Issue by kostja** [tarantool/tarantool#386]
Remove fiber.create(), fiber.resume(), fiber.detach() - these methods are confusing and most people use them to get the same effect as fiber.wrap(). So fiber.wrap() should be used instead.
Remove fiber.yield(). fiber.sleep(0) should be used instead.
fiber.status(f) should work also as a method: f:status().
